app/models/early_career_payments/slug_sequence.rb

```diff
@@ -14,6 +14,7 @@ class SlugSequence
       "supply-teacher",
       "entire-term-contract",
       "employed-directly",
+      "formal-performance-action",
```

Comment:
Just had a check against the v7 prototype and it looks like the slug is called "poor-performance", so I think this will need to be changed to match up to the prototype

Reply:
Hi we changed it from poor performance to stick to what its called on the previous journeys to stay consistent with the codebase